tests/devices/test_default_qutrit.py

```diff
@@ -72,6 +72,152 @@ class TestApply:
     """
 
     # TODO: Add tests for non-parametric ops after they're implemented
+    test_data_no_parameters = [
```

Comment:
Are these tests that were missing from the previous PR? The ones that would have been removed for inclusion with `expand_matrix`?

Reply:
No these aren't. All tests pertaining to `expand_matrix` would've been in `tests/test_operations.py`